vtStorAtaProtocol/vtStorAtaProtocol.pro

```diff
@@ -0,0 +1,56 @@
+#-------------------------------------------------
+#
+# Project created by QtCreator 2015-11-08T16:48:07
+#
+#-------------------------------------------------
+
+TARGET = vtStorAtaProtocol
+TEMPLATE = lib
+QMAKE_CXXFLAGS += -std=c++11
+QMAKE_CXXFLAGS += -fpermissive
+
+DEFINES += VTSTORATAPROTOCOL_LIBRARY
+
+SOURCES += \
+    AtaProtocolEssense1.cpp \
+    Platform/Linux/ProtocolAtaPassThrough.cpp
+
+HEADERS += \
+    AtaProtocolEssense1.h \
+    Platform/Linux/vtStorAtaProtocolPlatformDefines.h \
+    Platform/Linux/ProtocolAtaPassThrough.h
+
+symbian {
+    MMP_RULES += EXPORTUNFROZEN
+    TARGET.UID3 = 0xE701A033
+    TARGET.CAPABILITY = 
+    TARGET.EPOCALLOWDLLDATA = 1
+    addFiles.sources = vtStorAtaProtocol.dll
+    addFiles.path = !:/sys/bin
+    DEPLOYMENT += addFiles
+}
+
+unix:!symbian {
+    maemo5 {
+        target.path = /opt/usr/lib
+    } else {
+        target.path = /usr/lib
+    }
+    INSTALLS += target
+}
+
+INCLUDEPATH += "../../vtStorAtaProtocol/Platform/Linux" "../../Common" "../../Common/Platform/x86x64" "../../Common/Platform/x86x64/Linux" "../../StorageUtility" "../../StorageUtility/Linux" "../../vtStor" "../../vtStor/Platform/Linux"
+
+
+unix:!macx:!symbian: LIBS += -L$$PWD/../Build_StorageUtility/Debug/ -lStorageUtility
```

Comment:
This is path of library (dependency)
The path point to Debug folder in order to get library dependency
I building with Debug mode.